Tests/Mobile/NetworkObserverTests.swift

```diff
@@ -77,19 +78,22 @@ class NetworkObserverTests: ProcedureKitTestCase {
     }
 
     func test__network_indicator_only_changes_once_when_multiple_procedures_start() {
-        let procedure1 = TestProcedure()
+        let procedure1 = TestProcedure(delay: 1.0)
```

Comment:
Really minor quibble, but can we reduce these delays a couple of orders of magnitude? Just trying to keep the execution time of the tests down.